common/lib/xmodule/xmodule/assetstore/__init__.py

```diff
@@ -57,12 +57,12 @@ class AssetMetadata(object):
     EXPORTED_ASSET_FILENAME = u'assets.xml'
 
     @contract(asset_id='AssetKey',
-              pathname='basestring|None', internal_name='basestring|None',
-              locked='bool|None', contenttype='basestring|None',
-              thumbnail='basestring|None', fields='dict|None',
-              curr_version='basestring|None', prev_version='basestring|None',
-              created_by='int|long|None', created_by_email='basestring|None', created_on='datetime|None',
-              edited_by='int|long|None', edited_by_email='basestring|None', edited_on='datetime|None')
+              pathname='str|None', internal_name='str|None',
```

Comment:
decorator is not accepting six.string_types. I will check the documentation of contracts.